File: taskmanager_http/src/main/java/com/aglushkov/taskmanager_http/loader/http/HTTPConnectionBytesReader.java
package com.aglushkov.taskmanager_http.loader.http;

import com.example.alexeyglushkov.streamlib.handlers.ByteArrayHandler;

/**
 * Created by alexeyglushkov on 26.11.15.
 */
public interface HTTPConnectionBytesReader<T> extends ByteArrayHandler<T>, HTTPConnectionHandler {
}

File: taskmanager_httptask/src/main/java/com/aglushkov/taskmanager_http/loader/http/HTTPConnectionBytesReader.kt
package com.aglushkov.taskmanager_http.loader.http

import com.example.alexeyglushkov.streamlib.handlers.ByteArrayHandler

/**
 * Created by alexeyglushkov on 26.11.15.
 */
interface HTTPConnectionBytesReader<T> : ByteArrayHandler<T>, HTTPConnectionHandler